src/Modalize.tsx

```diff
@@ -119,6 +120,18 @@ export default class Modalize extends React.Component<IProps, IState> {
     this.onAnimateClose();
   }
 
+  public scrollTo = (option: {
+    y: number,
+    animated: boolean,
+  }): void => {
+    if (this.scrollView.current) {
+      const ScrollViewRef = (this.scrollView.current as any).getNode();
```

Comment:
If I am not mistaken, `.getNode()` would actually returns `_component` . I prefer `.getNode()` since accessing `_component` is essentially accessing the pricate properties of the component.

Reply:
I have now added an example of this in the `SnappingList` Modal. Let me know if it works.